### ISSUE TITLE
Panel: Panel header text shows multiple times on screen reader

### ISSUE THREAD

**Issue by mslonli** [OfficeDev/office-ui-fabric-react#7826]
<img width="505" alt="1548704596 1" src="https://user-images.githubusercontent.com/29870938/51861784-ef5fb380-22f1-11e9-8fa7-c8dbab235038.png">

<img width="505" alt="1548704060 1" src="https://user-images.githubusercontent.com/29870938/51861792-f25aa400-22f1-11e9-9b12-e59ee31ed9ba.png">

### Environment Information 

Using NVDA with Chrome
 
### How to reproduce
https://developer.microsoft.com/en-us/fabric#/components/panel
Use Panel - Small Panel, Full Screen, Fluid Width as an example

After look into the code, I am confused about the logic of this [piece of code
](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx#L163).
Should it be hidden when !isOpen || isAnimating?
 
#### Actual behavior: 
 
The panel header text shows three times
 
#### Expected behavior:
 
The panel header text shows only once

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) No
 
Requested priority: (Blocking, High, Normal, Low)  Normal
 
Products/sites affected: (if applicable)

**Comment by waabid** [comment]
Is there any update on this @natalieethell?

**Comment by natalieethell** [comment]
@mslonli @waabid sorry for the delay here! I've been discussing this issue with a couple folks on the team. In short, we're not sure of the fix yet, but are discussing potential changes to DOM structure and/or attributes like "role" on Panel.

**Comment by waabid** [comment]
Sounds good! Please let us know if we can help in anyway.

**Comment by natalieethell** [comment]
@waabid awesome, thank you! If you have an idea for a fix, definitely feel free to reach out and/or open a PR with the change.

**Comment by natalieethell** [comment]
I tried debugging this issue a little more by abstracting out the Panel control and found something interesting. Not sure if it'll solve the problem, but I'll put what I found here hoping that it maaaay lead us in the right direction.

I have three button + dialog combos in the following CodePen, each with different screen reader behaviors: https://codepen.io/naethell/pen/PyYwqN. 

The following descriptions apply specifically to Narrator:

* Dialog with a close button and aria-labelledby

  When opening the dialog, Narrator reads out the header text, the text in the close button, and then the 
header text again.

  **header text dialog, Close, button,
dialog, heading level 2 header text**

* Dialog with aria-labelledby and without a close button

  When opening the dialog, Narrator reads out nothing.

* Dialog with a close button and no aria-labelledby

  When opening the dialog, Narrator reads out the header text and the text in the close button once.

  **Close, button,
heading level 2 header text**


I think that the behavior above might have something to do with how a dialog gains focus. When a dialog contains a button, that button gains focus and other contents of the panel are read out, even if aria-labelledby is not specified. When a dialog does not contain a button, focus remains outside of the dialog, even if aria-labelledby is specified.
